oshc/main/static/main/css/app.css

```diff
@@ -1,241 +1,372 @@
 /*   GLOBAL DEFINATIONS STARTS  */
 
 * {
-  padding: 0;
-  margin: 0;
-  box-sizing: border-box;
-  -webkit-box-sizing: border-box;
-  -moz-box-sizing: border-box;
-  transition: opacity 0.7s ease-in-out;
-  font-family: "Montserrat", sans-serif
+    padding: 0;
+    margin: 0;
```

Comment:
Most of the changes in this file are related to indentation. Please undo these indentation changes. It's preferable to use a tab size of 2 spaces in `css` files.